### ISSUE TITLE
Collapse threads in the middle, not the end

### ISSUE THREAD

**Issue by ShadowJonathan** [gabboman/wafrn#67]
Just a suggestion, but when viewing threads in someone's page, it currently looks like this:

- A (OP)
- B
- C
- A
- D
- "Read more"

instead of that, why not collapse it like this?

- A (OP)
- "Expand"
- D
- A
- B
- A (latest)

**Comment by gabboman** [comment]
the collapse of threads is tumblr style. Might be a good idea to make an alternative view at some point, but right now this is the intended design.

Once I have more time I could try making some tests

**Comment by breadguyyy** [comment]
i agree with this one, maybe a setting to just show the last reply after the read more
